src/mongocxx/test/options/create_collection.cpp

```diff
@@ -1,9 +1,12 @@
 #include "catch.hpp"
 
+#include <bsoncxx/builder/stream/document.hpp>
+#include <bsoncxx/builder/stream/helpers.hpp>
 #include <bsoncxx/document/value.hpp>
 #include <bsoncxx/document/view.hpp>
 #include <bsoncxx/types.hpp>
 #include <bsoncxx/types/value.hpp>
+
```

Comment:
No newline (same as above).
Should the line below this be moved up above bsoncxx includes?